airbyte-commons-protocol/src/main/java/io/airbyte/commons/protocol/AirbyteMessageMigrator.java

```diff
@@ -75,6 +75,10 @@ public <PreviousVersion, CurrentVersion> CurrentVersion upgrade(final PreviousVe
     return (CurrentVersion) result;
   }
 
+  public Version getMostRecentVersion() {
+    return new Version(mostRecentVersion, "0", "0");
```

Comment:
Nit:  extract "0" to a named constant to make it easy to understand what it represents.

Reply:
I renamed `mostRecentVersion` to `mostRecentMajorVersion` which is actually what this attribute is. This should make it clearer already.
If you think it still helps to have the other constants, happy to change it as well.